install/update.native.php

```diff
@@ -296,11 +296,26 @@ function pluginGlpiinventoryUpdateNative($current_version, $migrationname = 'Mig
             INNER JOIN (
               SELECT
                 `computers_id`,
-                `last_inventory_update`
+                `last_inventory_update`,
+                `last_boot`
               FROM `glpi_plugin_glpiinventory_inventorycomputercomputers`
           ) AS `plugin_computers` ON `plugin_computers`.`computers_id` = `computers`.`id`
           SET
-              `computers`.`last_inventory_update` = `plugin_computers`.`last_inventory_update`
+          `computers`.`last_inventory_update` = `plugin_computers`.`last_inventory_update`,
+          `computers`.`last_boot` = `plugin_computers`.`last_boot`
```

Comment:
Mybe this should be done with two different queries to be able to add a where clause:
`` WHERE `computers`.`last_inventory_update` IS NULL``
`` WHERE `computers`.`last_boot` IS NULL``